MotionPhotoReader/app/src/androidTest/java/com/google/android/libraries/motionphotoreader/MotionPhotoReaderTest.java

```diff
@@ -32,18 +35,35 @@
  */
 @RunWith(AndroidJUnit4.class)
 public class MotionPhotoReaderTest {
-    private static final String filename = "MVIMG_20200621_200240.jpg";
-    private static final int NUM_FRAMES = 44;
+
+    private static final int NUM_FRAMES = 43;
     private static final long SEEK_AMOUNT_US = 10_000L;
-    
+
     private Context context;
+    private String[] testMotionPhotosList;
+    private String filename;
 
     /** A list of opened motion photo readers to close afterwards. */
     private final List<Runnable> cleanup = new ArrayList<>();
 
+    @Rule
+    public ActivityTestRule<MainActivity> activityRule =
+            new ActivityTestRule<>(
+                    MainActivity.class,
+                    /* initialTouchMode = */ true,
+                    /* launchActivity= */ true
+            );
+
     @Before 
     public void setUp() {
-        context = InstrumentationRegistry.getInstrumentation().getContext();
+        context = activityRule.getActivity().getApplicationContext();
+        AssetManager assetManager = context.getAssets();
+        try {
+            testMotionPhotosList = assetManager.list("motionphotos");
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
I think in this case, you'll want to throw the exception and not suppress it right ? Maybe have setup throws IOException ?
```
public void setUp throws IOException() {
   ...
   testMotionPhotosList=assetManager.list("motionphotos");
}
```

If you do want to suppress the exception, I would prefer:
```
 Log.e(TAG, "Error while setting up test", e);
```
over e.printStackTrace.

Reply:
Yeah, throwing it makes more sense. Changed.